3.6.1/Dockerfile

```diff
@@ -0,0 +1,137 @@
+#
+# Dockerfile for Python
+#
+FROM ubuntu:16.04
```

Comment:
Thanks for kind tips, please see below
Badness from Library Python image 
1.	It is also based from Linux, 
a.	https://github.com/docker-library/python/blob/d3c5f47b788adb96e69477dadfb0baca1d97f764/3.6/jessie/slim/Dockerfile 
2.	They have vulnerable components 
    a.	https://hub.docker.com/r/library/python/tags/3.6.2-slim/ 

Goodness from Ubuntu image
1.	We could have a full of control of it and customized programming easier
a.	Version upgrade 
b.	Environment parameter modification
2.	None business of copyright
3.	Avoid unnecessary components, keep the image tidy.